<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filtry_Liniowe</class>
 <widget class="QWidget" name="Filtry_Liniowe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="maska">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>361</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtr:</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>221</width>
     <height>276</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radio_zeruj">
      <property name="text">
       <string>zera</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_rozmyw">
      <property name="text">
       <string>Rozmywający</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_dolno">
      <property name="text">
       <string>Dolnoprzepustowy</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_gauss">
      <property name="text">
       <string>Rozmycie Gaussowskie</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_gorno">
      <property name="text">
       <string>Górnoprzepustowy</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_piw0st">
      <property name="text">
       <string>maska Prewitta 0 stopni</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_piw90st">
      <property name="text">
       <string>maska Prewitta 90 stopni</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_sob0st">
      <property name="text">
       <string>maska Sobela 0 stopni</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_sob90st">
      <property name="text">
       <string>maska Sobela 90 stopni</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_plasko">
      <property name="text">
       <string>Efekt plaskorzeźby</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>214</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Rozmiar maski:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSpinBox" name="spinBox_wartMaski">
      <property name="maximum">
       <number>9</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pCzysc">
      <property name="text">
       <string>Czysc maske</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="liczFiltr">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Włącz Filtr</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pOk">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pAnuluj">
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
